### PR TITLE
Make JIRA_PASSWORD an environment variable

### PR DESCRIPTION
**Pull request by MMore**
Maybe this could be a start for #8

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -34,7 +34,6 @@ type metricConfiguration struct {
 type configuration struct {
 	BaseURL     string                `yaml:"baseURL"`
 	Login       string                `yaml:"login"`
-	Password    string                `yaml:"password"`
```
Comment (zerok) [inline main.go:37]: Please keep the password still in there but set it to `yaml:"-"` :)

File: main.go [hunk 2]
```diff
@@ -193,7 +197,7 @@ func main() {
 		defer wg.Done()
 	}()
 
-	go check(ctx, log, cfg, &wg)
+	go check(ctx, log, cfg, &wg, jiraPassword)
```
Comment (zerok) [inline main.go:200]: The password should still be part of the configuration :)

File: main.go (hunk 1 above)
Comment (MMore) [reply main.go:37]: Ah well, makes sense :)

File: main.go [hunk 3]
```diff
@@ -170,6 +170,11 @@ func main() {
 		log.Fatal("Please specify a config file using --config CONFIG_FILE")
 	}
 
+	jiraPassword := os.Getenv("JIRA_PASSWORD")
```
Comment (zerok) [inline main.go:173]: `jiraPassword` still has to be added to the cfg struct :)

Comment (MMore) [reply main.go]: Sorry @zerok. You shouldn't work on those things when you're distracted. Should be fine now I hope.

**Pull request merged**